jetcd-launcher/src/main/java/io/etcd/jetcd/launcher/Etcd.java

```diff
@@ -89,6 +85,10 @@ public Builder withNetwork(Network network) {
         }
 
         public EtcdCluster build() {
+            String imagePro = System.getenv("ETCD_IMAGE");
+            if (!Strings.isNullOrEmpty(imagePro)) {
+                image = imagePro;
```

Comment:
this should be done in the constructor, otherwise it would override any call to `withImage`

Reply:
@lburgazzoli PTAL,Thanks.